vendor/sigs.k8s.io/gateway-api/apis/v1beta1/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
any idea why gateway-api deps are referenced now?

Reply:
https://github.com/knative-extensions/net-certmanager/blob/main/go.mod#L91 It is on my todo list to check why there was the dep there.

Reply:
It is a direct dep here: https://github.com/cert-manager/cert-manager/blob/master/go.mod#L50